frontend_app_file/src/container/PopupCreateFile.jsx

```diff
@@ -52,10 +52,17 @@ class PopupCreateFile extends React.Component {
   }
 
   componentWillUnmount () {
-    // console.log('%c<File> will Unmount', `color: ${this.state.config.hexcolor}`)
     document.removeEventListener(CUSTOM_EVENT.APP_CUSTOM_EVENT_LISTENER, this.customEventReducer)
   }
 
+  componentDidUpdate (prevProps, prevState, snapshot) {
```

Comment:
did you used snapshot at some points ?
the component doesn't implement getSnapshotBeforeUpdate so snapshot will be undefined